x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -364,7 +365,9 @@ private FunctionDefinition[][] functions() {
                 def(MvSlice.class, MvSlice::new, "mv_slice"),
                 def(MvZip.class, MvZip::new, "mv_zip"),
                 def(MvSum.class, MvSum::new, "mv_sum"),
-                def(Split.class, Split::new, "split") } };
+                def(Split.class, Split::new, "split") },
+            // Operators
+            new FunctionDefinition[] { def(In.class, In::new, "in") } };
```

Comment:
Does this mean you can use it like `WHERE IN(blah, "foo", "bar", "baz")` now? Do we want that?

Reply:
Probably best to zap this from `EsqlFunctionRegistry` then.